lib/generators/spire/install_generator.rb

```diff
@@ -0,0 +1,43 @@
+module Spire
```

Comment:
@caseyli I think this would warrant a version update. Not sure if its x, y, or z in the X.Y.Z version number.